Governance.md

```diff
@@ -0,0 +1,78 @@
+# Governance of the rules_scala
+
+These bazel rules for scala are under the bazelbuild org but governed by a
+contributors from the community.
+
+## Goals of this document
+
+Clarify:
+0. values of the project.
+1. how to get a PR merged
+2. how long you should need to wait for a review
+3. how to resolve conflicts in project direction
+4. how to make larger and disruptive changes
+
+## Values of rules_scala
+
+1. Correctness: bazel values fully reproducible builds which are reliable and hermetic (only depend on the current state of your repository)
```

Comment:
How about:
`Correctness: Bazel values fully reproducible builds which are reliable and hermetic (only depend on the current state of your repository). The default behavior of rules_scala is to strongly adhere to this philosophy. Deviations from correctness are only acceptable under opt-in configuration flags and should not exist at the expense of correct functionality.`

Reply:
I have to say I disagree. I think that maybe we should indeed add accessibility (as number 4) and there to say that until we find a solution that doesn't contradict the above values we're willing to live with existing solutions as long as they are opt-in...

Reply:
"Accessibility" sounds like a nice way to phrase it to me, +1.